.gitignore

```diff
@@ -1,3 +1,5 @@
+env/
+activate/
```

Comment:
Can you say more about these additions? Probably happy to include them. What generates these?